src/components/voltage-init-result.js

```diff
@@ -116,7 +114,7 @@ const VoltageInitResult = ({ result, status, tabIndex, setTabIndex }) => {
         fetchVoltageInitResult(studyUuid, currentNode.id).then((res) => {
             setResultToShow(res);
         });
-    }, [viNotif, disabledApplyModifications, studyUuid, currentNode.id]);
+    }, [disabledApplyModifications, studyUuid, currentNode.id]);
```

Comment:
maybe we should add a trigger on the `status` here ?

Reply:
but if status is always IDLE for example, that mean we will have a infite loop ?

Reply:
sorry you are right , the status is not supposed to change to many times

Reply:
result update after a status change will be manage by useNodeData in voltage-init-result-tab.js by the way, here it's to update result after the modification application (disabledApplyModification).